{
is_ring_available: true, 
# 1 for ouster LiDAR, 2 for velodyne LiDAR
lidar_type: 2, 

scan_frequency: 10,

lidar_vfov: 30.0, 
lidar_vfov_upper: 15, 
lidar_vfov_lower: -15, 

lidar_hfov: 360.0, 
lidar_lines: 16, 
lidar_hresolution: 900, 

blind: 0.5, 
}